### PR TITLE
Added smooth Navigation bar template

### PR DESCRIPTION
**Pull request by DeveloperAshish8**
I had added an animated Navigation Bar containing Logo and 6 anchor tags. 
I am a hacktoberfest participant , please merge the PR.

### CODE REVIEW

**Review by mudit023 — COMMENTED** [review]
Thanks for the contribtuition. Can you add a gif so that people can see the animation you have used?

File: Navigation Bar Template/Readme.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+Users should be able to:
+
+- Navigation bar containing 6 links (can be updated later).
+- Gives smooth animation on hover.
+
+### Screenshot
+
+![](<./Images/Screenshot%20(9).png>)
+
```
Comment (mudit023) [inline Navigation Bar Template/Readme.md:22]: Add the gif here in the Readme file.

**Comment by DeveloperAshish8** [comment]
Done @mudit023

**Comment by Susaksham** [comment]
> Done

> Done @mudit023

Please add the comments

**Comment by DeveloperAshish8** [comment]
In both HTML and CSS files?

**Comment by Susaksham** [comment]
> Done @mudit023

You haven't added gif in the Readme file . Please check again and add comments in both html and css so that it can become easier for others to understand

**Comment by DeveloperAshish8** [comment]
All done @Susaksham , have a look now.

**Review by Susaksham — APPROVED** [review]
Thanks for the contribution.

**Pull request merged**